models/bancor/ethereum/bancor_ethereum_sources.yml

```diff
@@ -0,0 +1,134 @@
+version: 2
+
+sources: 
+  - name: bancor_ethereum
+    description: "Ethereum decoded tables related to Bancor Network contract"
+    tables:
+      - name: BancorNetwork_v6_evt_Conversion
```

Comment:
Currently this table does not exist in v2, we may need to use other equivalent tables in v2 for this model.

Reply:
I am not sure but you can try to find some similar event table by searching `bancor` in v2 engine, and filter by chain and event type, like `bancor_ethereum.BancorConverter_evt_Conversion` and `bancor_ethereum.StandardPoolConverter_evt_Conversion`.

Reply:
maybe you wanted to refer to this one: `bancornetwork_ethereum.BancorNetwork_v6_evt_Conversion` ?

Reply:
> maybe you wanted to refer to this one: `bancornetwork_ethereum.BancorNetwork_v6_evt_Conversion` ?

Yeah. The first time when I committed the reference is right and I changed to the wrong one. Will fix it later